mypy/main.py

```diff
@@ -205,6 +206,36 @@ def invert_flag_name(flag: str) -> str:
     return '--no-{}'.format(flag[2:])
 
 
+if sys.platform == 'win32':
+    python_executable_prefix = 'py -{}'
+else:
+    python_executable_prefix = 'python{}'
+
+
+def _python_version_from_executable(python_executable: str) -> Tuple[int, int]:
+    try:
+        check = subprocess.check_output(python_executable.split(' ') +
```

Comment:
Not gonna fly if the path contains spaces.

Is `python_executable` allowed to contain spaces? I'd argue that `py -2` is not a valid executable anyway - it's just the tool you use to get hold of the real one.

Reply:
Hm, yes I think I will replace this with just a subprocess call to the `py -X.Y`, that should simplify things significantly.

Reply:
Looks like you got there first - glad we're on the same page!